src/server/services/generate-arpa-report.js

```diff
@@ -179,6 +179,7 @@ async function generateProject19 (records) {
             record.content.Name,
             record.content.Project_Identification_Number__c,
             record.content.Completion_Status__c,
+            record.content.Cancellation_Reason__c,
```

Comment:
Found this bug while I was testing the output: Notice in the output screenshots that the pre-fix 1.9 projects report has all the data shifted over one column, but it's corrected in the post-fix screenshot